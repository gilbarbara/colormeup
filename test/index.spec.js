import React from 'react';
import { AppConstants, ColorConstants, UserConstants } from 'constants/index';
import '../app/scripts/index';

jest.mock('redux-persist/lib/integration/react', () => ({
  PersistGate: () => (<div />),
}));

describe('Constants:AppConstants', () => {
  it('should match the snapshot', () => {
    expect(AppConstants).toMatchSnapshot();
  });
});

describe('Constants:ColorConstants', () => {
  it('should match the snapshot', () => {
    expect(ColorConstants).toMatchSnapshot();
  });
});

describe('Constants:UserConstants', () => {
  it('should match the snapshot', () => {
    expect(UserConstants).toMatchSnapshot();
  });
});
