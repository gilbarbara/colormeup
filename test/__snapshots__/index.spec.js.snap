// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants:AppConstants should match the snapshot 1`] = `
Object {
  "HIDE_ALERT": "HIDE_ALERT",
  "SHOW_ALERT": "SHOW_ALERT",
  "TOGGLE_SIDEBAR": "TOGGLE_SIDEBAR",
}
`;

exports[`Constants:ColorConstants should match the snapshot 1`] = `
Object {
  "INITIALIZE": "INITIALIZE",
  "SET_COLOR": "SET_COLOR",
  "SET_OPTIONS": "SET_OPTIONS",
}
`;

exports[`Constants:UserConstants should match the snapshot 1`] = `
Object {
  "REMOVE_COLOR": "REMOVE_COLOR",
  "RESET_USER_DATA": "RESET_USER_DATA",
  "SAVE_COLOR": "SAVE_COLOR",
  "SET_USER_OPTIONS": "SET_USER_OPTIONS",
}
`;
